src/main/java/com/powsybl/openloadflow/network/impl/extensions/OverloadManagementSystem.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network.impl.extensions;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface OverloadManagementSystem {
```

Comment:
Maybe just `OverloadSystem`?

Reply:
Overload system would be the system that trigger an overload. Here it is the system that manage or solve the overload.

Reply:
ok